bucket/libreoffice.json

```
@@ -21,13 +21,13 @@
         "# keeping files in 'LibreOffice' directory so that bundled font can work properly",
         "ensure \"$dir\\LibreOffice\\share\\fonts\" | Out-Null",
         "# usage: get_config ($name, $default). $default ==> set to what value if the value is empty (not being set before)",
-        "if (get_config 'MSIEXTRACT_USE_LESSMSI' $false) {",
-        "    # if MSIEXTRACT_USE_LESSMSI is true",
+        "if (get_config 'USE_LESSMSI' $true) {",
```

```suggestion
        "if (get_config USE_LESSMSI $false) {",
```
I know it's a bit confusing but by running the `get_config` command with `$true`, it creates the config with value `$true` if the config doesn't exist - we should keep it `$false` (i.e. the default). We can also get rid of the quotes because of the new convention in Scoop 0.3.0 (snake_case in config file and SCREAMING_CASE in code)